From 97b218e0213a0c54100301b0d32235b80fb41aac Mon Sep 17 00:00:00 2001
From: AndroPlus <hosigumayuugi@gmail.com>
Date: Sat, 31 Jul 2021 15:21:21 +0900
Subject: [PATCH 1/1] Remove su

---
 config/common.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index d79b8568..15df37b1 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -176,12 +176,6 @@ PRODUCT_PACKAGES_DEBUG += \
 # Root
 PRODUCT_PACKAGES += \
     adb_root
-ifneq ($(TARGET_BUILD_VARIANT),user)
-ifeq ($(WITH_SU),true)
-PRODUCT_PACKAGES += \
-    su
-endif
-endif
 
 # Dex preopt
 PRODUCT_DEXPREOPT_SPEED_APPS += \
-- 
2.25.1

